### PR TITLE
Fixes IV Drip Runtime

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes a runtime involving attaching IV drips to non-carbons.

also fixes https://github.com/ParadiseSS13/Paradise/issues/7489

### CODE REVIEW

File: code/game/machinery/iv_drip.dm [hunk 1]
```diff
@@ -1,10 +1,14 @@
+#define IV_TAKING 0
+#define IV_INJECTING 1
+
 /obj/machinery/iv_drip
 	name = "\improper IV drip"
 	icon = 'icons/obj/iv_drip.dmi'
 	icon_state = "iv_drip"
-	anchored = 0
+	anchored = FALSE
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
```
Comment (SamHPurp) [inline code/game/machinery/iv_drip.dm]: Could you expand on what this does, for my own understanding?

Comment (Fox-McCloud) [inline code/game/machinery/iv_drip.dm]: It ensures you see the little mouse dragging cursor.

Comment (SamHPurp) [inline code/game/machinery/iv_drip.dm]: Oh, simple as. Fair enough thanks.

Comment (Fox-McCloud) [inline code/game/machinery/iv_drip.dm]: http://www.byond.com/docs/ref/info.html#/atom/var/mouse_drag_pointer

**Pull request merged**